reporter/reporter.go

```diff
@@ -108,10 +110,21 @@ func (r *Reporter) Start() {
 	}
 	r.quitMu.Unlock()
 
-	r.bootstrapWithRetries(false)
+	r.bootstrapWithRetries()
+
+	if err := r.btcNotifier.Start(); err != nil {
+		r.logger.Errorf("Failed starting notifier")
+		return
+	}
+
+	blockNotifier, err := r.btcNotifier.RegisterBlockEpochNtfn(nil)
```

Comment:
Can we get the best known block after bootstrapping and set it as the parameter? Otherwise, the notifier will immediately get the chain tip which could be duplicate with the one processed during bootstrap